SofaKernel/extlibs/json/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.1)
+project(json C CXX)
+
+set(HEADER_FILES
+    nlohmann_json.h
+    json.h
+)
+
+
+add_library(${PROJECT_NAME} SHARED ${HEADER_FILES})
```

Comment:
@untereiner, why SHARED since this extlib is header only?

Reply:
I had an issue on os x compiling it as a static one. Can show it to you tomorow.